plugin/route53/route53.go

```diff
@@ -254,12 +254,55 @@ func (h *Route53) updateZones(ctx context.Context) error {
 				}
 				err = h.client.ListResourceRecordSetsPagesWithContext(ctx, in,
 					func(out *route53.ListResourceRecordSetsOutput, last bool) bool {
+						failoverList := map[string]map[string]*route53.ResourceRecordSet{}
 						for _, rrs := range out.ResourceRecordSets {
+							if rrs.SetIdentifier != nil {
```

Comment:
This will match every non-simple RR type not just failover. From docs:

```
Resource record sets that have a routing policy other than simple...
```